### DISCUSSION TITLE
Filepaths with spaces

### DISCUSSION THREAD

**Discussion by adrianlshaw** [netblue30/firejail#4472] [Q&A] [answered]
Hi there. I'm confining an application that expects a directory that has a name that includes spaces. 

I'm trying to use the command `whitelist ${HOME}/Foo Bar` but it's not working and it does not accept quote marks.

What is the syntax? Is this supported?

**Comment by adrianlshaw** [comment, marked as answer]
Must have made a typo, since the above seems to work after another try. Closing discussion.

**Reply by rusty-snake** [reply]
Yup, syntax is "just add spaces"


https://wiki.archlinux.org/title/firejail#Paths_containing_spaces:

> If you need to reference, whitelist, or blacklist a directory within a custom profile, such as with palemoon, you must do so using the absolute path, without encapsulation or escapes:
